src/detect.js

```diff
@@ -95,6 +95,7 @@ function getWebpackAssets(compilation) {
 
 function convertFilesToDict(assets) {
   return assets
+    .filter(file => file)
     .filter(file => file.indexOf("node_modules") === -1)
```

Comment:
I think it is better if we can do it in `filter(file => file && file.indexOf("node_modules") === -1)` ?

Reply:
I agree